dataplane/linux/bpf_ep_mgr.go

```diff
@@ -614,6 +614,8 @@ func (m *bpfEndpointManager) attachDataIfaceProgram(ifaceName string, polDirecti
 	epType := tc.EpTypeHost
 	if ifaceName == "tunl0" {
 		epType = tc.EpTypeTunnel
+	} else if ifaceName == "wireguard.cali" {
```

Comment:
nit: a switch perhaps?

Reply:
If you don't mind I'll leave this for now (to avoid another CI cycle), but will keep it in mind for further work.